demo/java/build.gradle.kts

```diff
@@ -13,8 +13,9 @@ repositories {
 
 dependencies {
     implementation(fileTree("../../source/kotlin/build/libs"))
-    // Or resolve it from jitpack like,
-    //   implementation("com.github.cosinekitty:astronomy:0.0.1")
+    implementation("org.jetbrains.kotlin:kotlin-stdlib:1.6.20") // Not needed if resolved from jitpack
+    // In an independent project resolve it from jitpack like,
+    //   implementation("com.github.cosinekitty:astronomy:x.y.z")
```

Comment:
I don't remember where exactly I saw this `x.y.z` but it seems like a better convention.